packages/rainbowkit/src/wallets/walletConnectors/walletConnectWallet/walletConnectWallet.ts

```diff
@@ -1,70 +1,24 @@
-import { Chain } from '../../../components/RainbowKitProvider/RainbowKitChainContext';
-import { getWalletConnectUri } from '../../../utils/getWalletConnectUri';
-import { isIOS } from '../../../utils/isMobile';
 import { Wallet } from '../../Wallet';
 import { getWalletConnectConnector } from '../../getWalletConnectConnector';
-import type {
-  WalletConnectConnectorOptions,
-  WalletConnectLegacyConnectorOptions,
-} from '../../getWalletConnectConnector';
-
-export interface WalletConnectWalletLegacyOptions {
-  projectId?: string;
-  chains: Chain[];
-  version: '1';
-  options?: WalletConnectLegacyConnectorOptions;
-}
 
 export interface WalletConnectWalletOptions {
   projectId: string;
-  chains: Chain[];
-  version?: '2';
-  options?: WalletConnectConnectorOptions;
```

Comment:
We still need to expose `options` here. I think we should keep `version` too but just mark the `1` as `@deprecated` with a typescript warning, and just remove any version checking/logic

Reply:
I think it'll be hard to mark version `1` as deprecated since they no longer expose `WalletConnectLegacyConnectorOptions` type. Will keep the `options` 🙏

Reply:
I suppose we could expose this upstream somewhere alongside a shared `projectId` config, if the WalletConnect Connector is shared. We may want to break that into a follow-up update though.

Reply:
Good idea yeah. Let me try that at least.